config/application.rb

```diff
@@ -47,7 +47,7 @@ class Application < Rails::Application
     config.autoload_paths += Dir["#{config.root}/lib/"]
 
     config.active_record.observers = :article_observer, :reaction_observer, :comment_observer
-    config.active_job.queue_adapter = :delayed_job
+    config.active_job.queue_adapter = :sidekiq
```

Comment:
This is actually covered by #6066, which will be deployed soon, so this won't even show up as a code diff.